AlexaSkillsKit.Lib/Helpers/DateTimeUtils.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Globalization;
+
+namespace AlexaSkillsKit.Helpers
+{
+    public class DateTimeUtils
+    {
+        public static DateTime FromAmazonRequest(string datetimeString)
+        {
+            try
+            {
+                DateTime datetime;
+
+                //Try a generic parse
+                if (DateTime.TryParse(datetimeString, CultureInfo.InvariantCulture, DateTimeStyles.None, out datetime))
+                {
+                    return datetime;
+                }
+
+                //Try a ticks parse
+                long ticks;
```

Comment:
"ticks" and "milliseconds" are two distinct concepts. The variable name here should be "milliseconds".